packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -170,28 +172,28 @@ function WorklistDataGrid(props) {
       <WorklistCellTag
         key={cellColumnIndex}
         {...tabIndex}
-        className={cx('worklist-data-grid-cell', { masked: cell.isMasked })}
+        className={cx('worklist-data-grid-cell', { masked: cell.isMasked, selectable: (!cell.isMasked && cell.isSelectable) })}
```

Comment:
Since the nature of this grid is that its cells are generally interactable, should isSelectable default to true?

Reply:
That is a great requirement question.  I did ask this question during the development of this JIRA.  The only reason that this was not made the default behavior was to remain consistent with the clinical data grid implementation.  If this is the desired behavior, we could definitely add that logic.  Do we need a new JIRA to hash that out?  I am happy to code it, if agreed upon.